gnu-apl-editor.el

```diff
@@ -11,7 +11,7 @@ the function and set it in the running APL interpreter."
   (interactive (list (gnu-apl--choose-variable "Function name" :function (gnu-apl--name-at-point))))
   (gnu-apl--get-function name))
 
-(defvar gnu-apl-flash-on-send t
+(defcustom gnu-apl-flash-on-send t
```

Comment:
The `defcustom` form does not set `:type` and `:group`. Look at the definition of `gnu-apl-auto-function-editor-popup` for an example of how it's done.